.github/workflows/vanilla_ubuntu.yml

```diff
@@ -0,0 +1,148 @@
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Configure AWS credentials
+        id: creds
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-gov-west-1
+
+      - name: Terraform Init
+        id: init
+        run: |
+          cd testing/
+          terraform init
+
+      - run: 'echo "$SSH_KEY" > .key'
+        shell: bash
+        env:
+          SSH_KEY: ${{secrets.SSH_PRIVATE_KEY}}
+
+      - run: chmod 400 .key
+
+      - name: Terraform private key
+        run: cp .key testing/.key
+
+      - name: Terraform Validate
+        id: validate
+        run: |
+          cd testing/
+          terraform validate -no-color
+
+      - name: Terraform Plan
+        id: plan
+        run: |
+          cd testing/
+          terraform plan -var "GITHUB_RUN_ID=$GITHUB_RUN_ID" -no-color
+        continue-on-error: true
+
+      - name: Terraform Plan Status
+        if: steps.plan.outcome == 'failure'
+        run: exit 1
+
+      - name: Terraform Apply
+        run: |
+          cd testing/
+          terraform apply  -var "GITHUB_RUN_ID=$GITHUB_RUN_ID" -auto-approve
+
+      - name: Set up Python
+        id: setup_python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip3 install ansible pytest-testinfra
+
+      - name: Ansible Version check
+        run: ansible --version
+
+      - name: Create inventory hosts.ini
+        run: |
+          touch hosts.ini
+          echo "[rke2_servers]" > hosts.ini
+          aws ec2 describe-instances --filters "Name=tag:Owner,Values=rke2-ansible-github-actions" "Name=tag:NodeType,Values=Server" "Name=tag:github_run,Values=$GITHUB_RUN_ID" --query "Reservations[*].Instances[*].PublicIpAddress" --output text >> hosts.ini
+          echo "[rke2_agents]" >> hosts.ini
+          aws ec2 describe-instances --filters "Name=tag:Owner,Values=rke2-ansible-github-actions" "Name=tag:NodeType,Values=Agent" "Name=tag:github_run,Values=$GITHUB_RUN_ID" --query "Reservations[*].Instances[*].PublicIpAddress" --output text >> hosts.ini
+          echo "" >> ansible.cfg
+          echo "" >> ansible.cfg
+          echo "remote_user=ubuntu" >> ansible.cfg
+          echo "private_key_file=.key" >> ansible.cfg
+
+      - name: Check hosts.ini and ansible.cfg
+        run: |
+          cat hosts.ini
+          cat ansible.cfg
+
+      - name: Prep inventory
+        run: |
+          cp -R inventory/sample inventory/ubuntu20
+          cp hosts.ini inventory/ubuntu20/hosts.ini
+
+      - name: Run playbook
+        run: |
+          ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -i inventory/ubuntu20/hosts.ini -u ubuntu --verbose --private-key .key site.yml
```

Comment:
@dgvigil - was looking at output of build job, and verbosity may be too high, as it took a bit of scrolling to see anything useful. Perhaps a var to support tuning it for future debugging purposes (i.e., `vv` as default, and `vvvv` if you need more details on subsequent runs?)... If this a lot of work, we can turn into issue and revisit on another date...

Reply:
I figured that since no one would actually be looking over the output unless there was an error, so the `--verbose` flag seemed fitting.  The `--verbose` flag equals `-v`.